blocks/card/card.css

```diff
@@ -0,0 +1,86 @@
+.card-container {
+  container-type: inline-size;
+  container-name: card-container;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #dadada;
```

Comment:
This is `--spectrum-gray-300` on light and `--spectrum-gray-800` on dark

Reply:
Removed the border entirely! It's only a `box-shadow` here.